package.json

```diff
@@ -114,6 +115,7 @@
     "webpack-cli": "^4.2.0",
     "webpack-node-externals": "^1.7.2",
     "trim-newlines": "^3.0.1",
-    "css-what": "^5.0.1"
+    "css-what": "^5.0.1",
+    "glob-parent": "^5.1.2"
```

Comment:
I'm not sure if it's required to add these as dev dependencies. If it works without it, we should remove.

Reply:
Yup no problem once we confirm the convention we can delete the old obsolete code